app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<header class="home">
+<div class="toolbar">
+  <%= link_to image_tag("https://img.icons8.com/officel/344/back.png"),groups_path, class: "back_btn" %>
+  <h5 class="toolbar__groups">Groups</h5>
+</div>
+</header>
+
+<% if flash[:notice] %>
+  <div class="flash-notice">
+    <p><%= notice %></p>
+  </div>
+<% end %>
+
+<section class="main-container">
+  <div id="groups">
+    <% @groups.each do |group| %>
+      <%= render group %>
+    <% end %>
+  </div>
+  <div class="splash-buttons">
+    <%= link_to "New group", new_group_path %>
+  </div>
```

Comment:
Well done on the UI, The app is impressive!!! 👏🏾 

There are a few changes required by the previous reviewer that haven't been met yet. Kindly make sure that all the required changes are addressed to get approval.

### Design
- Thank you for making the "New group' and "New Transaction" buttons stick to the bottom as requested. Kindly implement the green background and inner white border design as required by the design guidelines.

The "New group' and "New Transaction" buttons should look like this:

![image](https://github.com/mohammadnewton/budget-app/assets/38088818/591b4714-3052-4879-ad42-2ce9a7da20d0)